log-manager/src/main/java/io/airlift/log/LoggingConfiguration.java

```diff
@@ -140,7 +140,11 @@ public String getLogAnnotationFile()
         return logAnnotationFile;
     }
 
-    @Config("log.annotation-file")
+    /* Reuse the same property as defined in io.airlift.node.NodeConfig because NodeInfo objects
+       are not yet available when the Logging is being configured.
+     */
+    @Config("node.annotation-file")
+    @LegacyConfig("log.annotation-file")
```

Comment:
I'm not sure if I should use `@Deprecated` here instead, would this raise an exception if the old name is used?

Reply:
The docs say that "diagnostics will be logged": https://github.com/airlift/airlift/blob/master/docs/ref_configuration.md#legacyconfig